cd-cli/cnf-templates/complete-pipeline.yaml

```diff
@@ -431,6 +431,12 @@ Resources:
                     sed -i -e "s|pn_portfat_imageUrl=.*|pn_portfat_imageUrl=$IMAGE_URL|" desired-commit-ids-env.sh
                     export PIPELINE_NAME="pn-pn-portfat-update-pipeline"
                   fi
+                - |
+                  if ( [ "$PROJECT" = "pn-delayer" ] ) then
+                    sed -i -e "s|pn_delayer_commitId=.*|pn_delayer_commitId=$COMMIT_ID|" desired-commit-ids-env.sh
+                    sed -i -e "s|pn_delayer_imageUrl=.*|pn_delayer_imageUrl=$IMAGE_URL|" desired-commit-ids-env.sh
+                    export PIPELINE_NAME="pn-delayer-update-pipeline"
```

Comment:
@vincenzo-racca-pa `pn-pn-delayer-update-pipeline`

Reply:
fixato il Name in PnDelayerDeployPipeline con `'${ProjectName}-delayer-update-pipeline'`